.spelling

```diff
@@ -1111,6 +1113,8 @@ wildcarded
 wildcards
 workgroup
 workload
+WorkloadEntry
+WorkloadGroup
```

Comment:
My first thought with these are would would have used backticks.

Reply:
And I'm not seeing them in the text changes below.

Reply:
I can revert them to backticks if that's preferred.  I did it this way because I noticed there were several other Istio objects in the .spelling file

Reply:
Our contribution guidelines say to use Backticks. We can always pick up later if needed.